### ISSUE TITLE
WLASL matching poses to examples

### ISSUE THREAD

**Issue by AmitMY** [gulvarol/bsl1k#4]
I'm looking at the WLASL dataset, which includes `21,083` "instances" of signs.
However, the `pose.pkl` for that dataset is a list of `20,245` items.

What is the correct way to map between videos and poses?

**Comment by gulvarol** [comment]
After our download procedure, the videos which were "readable" (according to this line https://github.com/gulvarol/bsl1k/blob/master/misc/wlasl/create_info.py#L111) were saved in the `info.pkl` which was released here: https://github.com/gulvarol/bsl1k/#supported-datasets
This `info.pkl` has the metadata for 20,245 videos which corresponds exactly to the poses. We were able to only use these videos in our experiments. Hope this helps.
